### PR TITLE
edit des projet ds un modal

### PR DESCRIPTION
**Pull request by Jonathan92ETS**
| Question               | Answer
| ---------------------- | -
| Bugs correction ?      | no
| New features ?         | yes
| Related issues         | #5

---

##### What have you changed  ?
...

##### How did you do that ?
...

### CODE REVIEW

File: tasklist/src/config/env.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-	"environment": "development"
+	"environment": "production"
```
Comment (RignonNoel) [inline tasklist/src/config/env.json]: please let the development mode by default in the repository

File: tasklist/src/pages/listProject/listProject.html [hunk 2]
```diff
@@ -23,6 +23,11 @@
       </ion-item>
 
       <ion-item-options side="right">
+        <button ion-button (click)="openEditModal(item)">
+          <ion-icon name="create"></ion-icon>
+          Edit
```
Comment (RignonNoel) [inline tasklist/src/pages/listProject/listProject.html]: Please use the same icon as the task edition to keep a good user experience.

**Review by RignonNoel — CHANGES_REQUESTED** [review]
Hi! Thx for this PR! 👍 

It seems that you are not using the same architecture as the one used for tasks edition. That would be surely a good point.

Anyway it seems work but there are some little details to be consistent.

**Comment by Jonathan92ETS** [comment]
- Ajout de la possibilité de modifié le projet a partir du menu de Task List
- Modification de l'icon
- Environment a development

File: tasklist/src/pages/listProject/listProject.html [hunk 3]
```diff
@@ -23,6 +23,11 @@
       </ion-item>
 
       <ion-item-options side="right">
+        <button ion-button (click)="openEditModal(item)">
+          <ion-icon name="brush"></ion-icon>
+          Edit
```
Comment (RignonNoel) [inline tasklist/src/pages/listProject/listProject.html]: Please check the edit button of task made by @kristofbc.
```
<ion-buttons end>
  <button ion-button icon-only (click)="editTapped($event)">
    <ion-icon name="brush"></ion-icon>
  </button>
</ion-buttons>
```

**Review by RignonNoel — CHANGES_REQUESTED** [review]
Please check the PR #25 to edit task made by @kristofbc . His architecture ust need to be copy/paste to project

File: tasklist/src/pages/listProject/listProject.html (hunk 3 above)
Comment (Jonathan92ETS) [reply tasklist/src/pages/listProject/listProject.html]: C'est deja copié... 
J'ai juste laisser l'autre aussi, mais je peut le supprimer si tu veut juste un point d entrer pour la modif

**Comment by RignonNoel** [comment]
I don't know if you see my general comment on the review so i just want to copy/paste here to you.

The PR work but you don't have the correct architecture to be consistent with the PR of Kristof, the PR #25 is an exemple of his works. A simple copy paste of some attribute seems be ok to redo this PR on project.

---
**1st review**

Hi! Thx for this PR! 👍
It seems that you are not using the same architecture as the one used for tasks edition. That would be surely a good point.
Anyway it seems work but there are some little details to be consistent.

---
**2nd review**
Please check the PR #25 to edit task made by @kristofbc . His architecture just need to be copy/paste to project

**Review by RignonNoel — CHANGES_REQUESTED** [review]
Yeah, seems good to me. 

You just rename your commit to fit with the max char length and we're in business ;)

**Review by RignonNoel — APPROVED** [review]

**Comment by RignonNoel** [comment]
I let you the pleasure to merge with the big green button ;)

**Pull request merged**